tools/block_cache_analyzer/block_cache_trace_analyzer.cc

```diff
@@ -1830,8 +1830,8 @@ void BlockCacheTraceAnalyzer::PrintDataBlockAccessStats() const {
             hist_naccess_per_key.Add(caller_access.second);
           }
         }
-        uint64_t avg_accesses = hist_naccess_per_key.Average();
-        uint64_t stdev_accesses = hist_naccess_per_key.StandardDeviation();
+        uint64_t avg_accesses = static_cast<unit64_t>(hist_naccess_per_key.Average());
+        uint64_t stdev_accesses = static_cast<unit64_t>(hist_naccess_per_key.StandardDeviation());
```

Comment:
This is a typo: `unit64_t`.

Reply:
Doh!
Sorry about that, I just fixed and pushed